BrickStructureLines lines
string frame_id
---
bool success
geometry_msgs/PoseStamped pose